### ISSUE TITLE
Was GitHub Management used as a template on purpose?

### ISSUE THREAD

**Issue by galargh** [protocol/atlas.storage#1]
Hey, I'm reaching out cause I noticed you used https://github.com/protocol/github-mgmt-template as a template for this repository. It's a project that aims to express GitHub configuration as code. If that was intentional, I'd be extremely interested in finding out more about intended use case and try to help you make the most out of it :)

**Comment by eshan17** [comment]
@galargh I don't think it was. I was just setting a new GitHub repo and didn't realize I used that template. I also did this a while ago, so I don't even remember how I set it up from the template 🙈
